### PR TITLE
Remove default select option from dropdown

### LINKED ISSUE #343: Remove default select option from plugins dropdown
Proposing to remove default select option:

`<option value=""><?php esc_html_e( 'Select Plugin', 'plugin-check' ); ?></option>`

Ref: `/templates/admin-page.php` L26

* With this number of clicks will be reduced when there is only one option in the dropdown.
* Since this is a required field, making one selected would be better UI I believe.

### PR DESCRIPTION
**Pull request by ernilambar**
Fixes https://github.com/WordPress/plugin-check/issues/343

### CODE REVIEW

**Comment by mukeshpanchal27** [comment]
Thanks @ernilambar for the PR. I like the idea to remove default empty selection. If we there is no plugin then it will show `No plugins available.` message.

I value the input from @felixarntz and @joemcgill on this matter.

**Comment by felixarntz** [comment]
Hmm, I'm not sure about this. The only benefit I could see here is if a site only has a single plugin and that's what you want to test. But otherwise I would find it an arbitrary choice to have the alphabetically first plugin pre-selected, so from that perspective I prefer the neutral default we have now.

How about this?
* If there's one plugin, we only put that plugin there and have it pre-selected.
* Otherwise, things remain as they are now.

I think that would be a good compromise that would address both concerns.

I don't feel strongly about it though, curious to hear other perspectives.

**Comment by mukeshpanchal27** [comment]
> How about this?
> 
> * If there's one plugin, we only put that plugin there and have it pre-selected.
> * Otherwise, things remain as they are now.

I personally like these suggestion.

**Comment by ernilambar** [comment]
PR updated as per the comment.

File: templates/admin-page.php [hunk 1]
```diff
@@ -23,7 +23,13 @@
 				</h2>
 
 				<select id="plugin-check__plugins-dropdown" name="plugin_check_plugins">
-					<option value=""><?php esc_html_e( 'Select Plugin', 'plugin-check' ); ?></option>
+					<?php
+					if ( 1 !== count( $available_plugins ) ) {
+					?>
```
Comment (mukeshpanchal27) [inline templates/admin-page.php]:
```suggestion
					<?php if ( 1 !== count( $available_plugins ) ) { ?>
```

File: templates/admin-page.php [hunk 2]
```diff
@@ -23,7 +23,13 @@
...
+						<option value=""><?php esc_html_e( 'Select Plugin', 'plugin-check' ); ?></option>
+					<?php
+					}
+					?>
```
Comment (mukeshpanchal27) [inline templates/admin-page.php]:
```suggestion
					<?php } ?>
```

**Review by mukeshpanchal27 — COMMENTED** [review]
Thanks for the update @ernilambar. Left nit-pick other then look good to me.

**Review by swissspidy — APPROVED** [review]

**Review by felixarntz — APPROVED** [review]
Thanks @ernilambar!

**Pull request merged**